packages/gitbeaker-cli/test/integration/index.ts

```diff
@@ -20,7 +20,7 @@ function cli(cmd, options = {}) {
 }
 
 beforeEach(() => {
-  env = {};
+  env = process.env;
```

Comment:
This would not reset it?

I think you need declare it once outside the anonymous function.

Reply:
I did this on purpose, because i modify the env var passed each time, i want to reset it before each test.